docs/machine-learning/tutorials/image-classification-api-transfer-learning.md

```diff
@@ -25,7 +25,7 @@ In this tutorial, you learn how to:
 
 ## Prerequisites
 
-- [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019) or later with the ".NET Core cross-platform development" workload installed.
+- [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/).
```

Comment:
I think we still need to add the workload. Based on the .NET tutorial for console apps, it seems ".NET Desktop Development" workload is required.

https://docs.microsoft.com/dotnet/core/tutorials/with-visual-studio?pivots=dotnet-6-0

Reply:
They should all now be updated.